src/app/zap-templates/zcl/data-model/chip/refrigerator-alarm.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <bitmap name="AlarmMap" type="BITMAP32">
+     <cluster code="0x0057" />
+     <item name="DoorOpen" value="0" mask="0x00"/>
+  </bitmap>
+  <cluster>
+    <name>Refrigerator Alarm</name>
+    <domain>Home &amp; Appliances</domain>
+    <description>Attributes and commands for configuring the Refrigerator alarm.</description>
+    <code>0x0057</code>
+    <define>REFRIGERATOR_ALARM</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <attribute side="server" code="0x0000" define="MASK" type="AlarmMap" default="0" writable="true"  optional="false">Mask</attribute>
+    <attribute side="server" code="0x0001" define="LATCH" type="AlarmMap" default="0" writable="false" optional="false">Latch</attribute>
+    <attribute side="server" code="0x0002" define="STATE" type="AlarmMap" default="0" writable="false" optional="false">State</attribute>
+
+    <command source="client" code="00" name="Reset" optional="false">
+        <description>Reset alarm</description>
+        <arg name="Alarms" type="AlarmMap" optional="false"/>
+        <arg name="Mask" type="AlarmMap" optional="true"/>
+    </command>
+
+    <event side="server" code="0x00" priority="info" name="Notify" optional="false">
```

Comment:
I don't see where the spec defines the event priority, but I guess OK for now.  Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7006